src/style.css

```diff
@@ -1,3 +1,83 @@
+:root {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+
+  --color-text: rgba(0, 0, 0, 1);
+  --color-activ: rgba(0, 172, 220, 1);
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  padding: 0 50px;
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.nav_list {
+  display: flex;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  margin: 0 10px;
```

Comment:
I asked you to use `:last-child ` pseudo-class to order to reset margin-right to 0 for the last element, but I've never recommended to set margin left & right for the element. It's actually a mistake here. 

Since it's a flex container, ideally, you should use 'gap' property to position elements. However, in this task margin is required instead of gap, so we have to use margins. 
But let's not make it worse: by using margin left & right you making it harder to check the spacing between elements and has to use two pseudo-classes (for the fists and last child) instead of just one. 

So let's use margin-right: 20 in combination with last-child pseudo-class. No need in adding extra complexity here.